src/internal/internal.go

```diff
@@ -75,21 +74,30 @@ func (g *fileGetter) Send(bytes *types.BytesValue) error {
 	return err // always nil
 }
 
-func GetPipelineDetails(ctx context.Context, apiServer pfs.APIServer, info *pps.PipelineInfo) error {
+type fileReader interface {
```

Comment:
All of these helper methods can go in `src/internal/clientsdk`

Reply:
I can definitely do that, I worried it would be confusing since they aren't really wrappers around client functions as these are. I also thought we might want to make the existing clientsdk functions non-internal at some point, since it would help anyone writing code against our API.

Reply:
Maybe ppsutil then?  I'm not really a fan of just dumping things into the root of `./src/internal`.